README.md

```diff
@@ -6,18 +6,14 @@ https://bitcoincore.org
 For an immediately usable, binary version of the Bitcoin Core software, see
 https://bitcoincore.org/en/download/.
 
-Further information about Bitcoin Core is available in the [doc folder](/doc).
-
-What is Bitcoin?
-----------------
+What is Bitcoin Core?
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
+Bitcoin Core connects to the Bitcoin P2P network to download and fully validate
+blocks and transactions. It also includes a wallet and graphical user
+interface, that can be optionally built.
```

Comment:
suggestion: "a wallet and a graphical user interface (GUI), both of which can be optionally built."

Reply:
Thanks, switched to "which"